Rakefile

```diff
@@ -25,16 +25,8 @@ namespace :test do
     task.pattern = 'test/**/*_test.rb'
   end
 
-  ExerciseTestTask.new :exercise do |task|
-    task.description = "Run the tests for a specific exercise"
-  end
-
   desc 'Run the tests for all exercises'
   task :exercises do
-    Dir.foreach('exercises').each do |exercise|
-      next if %w(. ..).include?(exercise)
-
-      Rake::Task['test:exercise'].execute(exercise: exercise)
-    end
+    ExerciseTests.run_all
```

Comment:
Do you think `ExerciseTests.run(FileList['exercises/*'].pathmap('%f'))` then?

I would really like there to be a "name" for `FileList['exercises/*'].pathmap('%f')`, but I'm not sure where to put it. `ExerciseTests.run(ExerciseTests.all)` doesn't feel right to me.

Thank you so much for all of your feedback!